docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -340,6 +340,8 @@ Noteworthy module changes
 Plugins
 =======
 
+* macOS no longer defaults to using the ``paramiko`` connection plugin as it has since Ansible 1.8. It will now use the ``ssh`` connection plugin by default.
```

Comment:
Since this is a porting guide entry, we probably need to be more prescriptive about how to keep things working on a Mac. Something like:

As `ssh` supports connection persistence between tasks and playbook runs, it allows much higher performance than paramiko. However, if using ssh password authentication, it will be necessary to install the `sshpass` utility (left as an exercise for the reader) or explicitly set the connection type to `paramiko`.

Reply:
I wasn't familiar enough with the differences to articulate it well. Thanks for this info. I will incorporate it.